__tests__/api.test.js

```diff
@@ -365,6 +365,32 @@ describe('PATCH /api/articles/:article_id', () => {
             expect(body.message).toBe('article_id does not exist');
           });
       });
+
+})
+describe('GET /api/users', () => {
+    test('Responds with status code 200 and with response message of all users', () => {
+    return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toHaveProperty('users');
+            });
+    });  
+    test("Data is in the correct format", () => {
+        return request(app)
+        .get('/api/users')
+        .then(({ body }) => {
+        const users = body.users;
+
+        expect(Array.isArray(users)).toBe(true);
+            users.forEach((user) => {
+            expect(Object.keys(user).length).toBe(3) 
+            expect(typeof user.username).toBe('string');
+            expect(typeof user.name).toBe('string');            
+            expect(typeof user.avatar_url).toBe('string');            
+        });
```

Comment:
Realistically, your second test achieves the same as your first test, but with more detail, so if you only had your second test, this would still be a complete test suite for this end point :)

Reply:
thankyou will delete and merge